ricecooker/sushi_bar_client.py

```diff
@@ -59,7 +61,7 @@ def send(self, data):
                 self.ws.send(data)
                 return
             except websocket.WebSocketConnectionClosedException:
-                print('WebSocket closed, retrying send.')
+                config.LOGGER.debug('WebSocket closed, retrying send.')
```

Comment:
Were you able to get it to run locally with this? I tried doing this, and it caused some sort of infinite loop (might be resolved now, but not sure)

Reply:
Haven't tried localhost no... I will keep an eye out for this before release to PyPI.

Maybe it's trying to log to the socket, and then remote-web-socket is failing which generates a new log message etc. I can see how this could be bad...